routers/repo/http.go

```diff
@@ -437,8 +438,17 @@ func (h *serviceHandler) setHeaderCacheForever() {
 	h.w.Header().Set("Cache-Control", "public, max-age=31536000")
 }
 
+// sendFile send the provided content from a file
 func (h *serviceHandler) sendFile(contentType, file string) {
-	reqFile := path.Join(h.dir, file)
+	reqFile := filepath.Join(h.dir, file)
+	rel, err := filepath.Rel(h.dir, reqFile)
+	if err != nil {
+		http.Error(h.w, "Detect if path available failed", 500)
+		return
+	} else if rel == ".." || strings.HasPrefix(filepath.ToSlash(rel), "../") {
+		http.NotFound(h.w, h.r)
+		return
+	}
```

Comment:
I'm not sure that this is strictly necessary anymore after the changes in GetTextFile but it's potentially good practice